test/wait_for_images.py

```diff
@@ -17,7 +17,7 @@ def check_image_exists(image):
     @retry(tries=60, delay=10)
     def check(name):
         subprocess.check_output(
-            ["docker", "manifest", "inspect", name],
+            ["docker manifest inspect " + name],
```

Comment:
?????

Reply:
The execution with `shell=true` seems to require this format. The old one will always pass. :/